File: Demo/Sources/ViewControllers/PageControl/PageControlViewController.m
//
//  Globus
//

#import "PageControlViewController.h"

@interface PageControlViewController ()

@property(nonatomic, weak) IBOutlet GLBPageControl* pageControl;

@end

@implementation PageControlViewController

#pragma mark - UIViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    _pageControl.tapBehavior = GLBPageControlTapBehaviorJump;
    _pageControl.numberOfPages = 10;
}

#pragma mark - Actions

- (IBAction)pressedMenu:(id)sender {
    [self.glb_slideViewController showLeftViewControllerAnimated:YES complete:nil];
}

@end

File: Demo/Sources/ViewControllers/PageControl/PageControlViewController.swift
//
//  Globus
//

import Globus

class PageControlViewController: GLBViewController {
    
    // MARK - Outlet property
    
    @IBOutlet fileprivate weak var pageControl: GLBPageControl!
    
    // MARK - UIViewController
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        self.pageControl.tapBehavior = .jump
        self.pageControl.numberOfPages = 10
    }
    
    // MARK - Action
    
    @IBAction internal func pressedMenu(_ sender: Any) {
        self.glb_slideViewController?.showLeftViewController(animated: true, complete: nil)
    }
    
    // MARK: - GLBNibExtension
    
    public override static func nibName() -> String {
        return "PageControlViewController-Swift"
    }
    
}
